### DISCUSSION TITLE
What if I my existed database has `users` table

### DISCUSSION THREAD

**Discussion by djyde** [supabase/supabase#1262] [Questions]
Hi. I am going to migrate my existed database to supabase. I found that in supabase, there is a table named `users`, which is used by authentication. But my database has a table named `users` too. 

When I tried to delete this table, it does not allow me to do it. 

What should I do?

**Comment by silentworks** [comment]
Do not delete this table, this table is inside of a private schema called `auth` and won't have any effect on your own `users` table as that would be created under the `public` schema. Rule of thumb, don't delete any table you didn't create under `auth`, `extensions` or `storage`. Supabase need these in order to work as expected.
